Google.PowerShell/Common/CloudSdkSettings.cs

```diff
@@ -94,10 +152,27 @@ public static string GetCurrentConfigurationFilePath()
 
         /// <summary>
         /// Returns the setting with the given name from the currently active gcloud configuration.
+        /// We first look into the user config file. If we cannot find anything, we look
+        /// into the global properties file.
         /// </summary>
         public static string GetSettingsValue(string settingName)
         {
-            string configFile = GetCurrentConfigurationFilePath();
+            string userConfigFile = GetCurrentConfigurationFilePath();
+            string userConfigSetting = GetSettingsValueFromFile(settingName, userConfigFile);
+
+            if (string.IsNullOrWhiteSpace(userConfigSetting))
+            {
+                userConfigSetting = GetSettingsValueFromFile(settingName, InstallationPropertiesPath);
+            }
+
+            return userConfigSetting;
+        }
+
+        /// <summary>
+        /// Retrieves the setting with the given name from a config file.
+        /// </summary>
+        private static string GetSettingsValueFromFile(string settingName, string configFile)
```

Comment:
This is a style nit, but personally I assume function parameters go from most important to least important. So putting the file name before the setting name seems more intuitive to me. But YMMV.